### ISSUE TITLE
DeprecationWarning durnig pytest-cov

### ISSUE THREAD

**Issue by Jakub89** [nameko/nameko#685]
We've updated nameko from stable to `3.0.0rc8` to get rid of pytest warnings. And we were successful, but one different warning appear:
```
========================================== warnings summary ==========================================
.../lib/python3.8/site-packages/nameko/messaging.py:38
  .../lib/python3.8/site-packages/nameko/messaging.py:38: DeprecationWarning: invalid escape sequence \.
    re.sub("^{}\.".format(prefix), "", key): value

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

**Environment:**
 - Nameko version: 3.0.0rc8
 - Python version: 3.8.2
 - OS: Manjaro Linux 64-bit GNOME 3.36.2
 - pytest: 5.4.1
 - pytest-cov: 2.8.1

**Comment by zsiciarz** [comment]
Looks like the regex string is missing `r` prefix. I proposed a fix in #690 .

**Comment by mattbennett** [comment]
Fixed in #690. Thanks @zsiciarz!
